### PR TITLE
Document new websphere-liberty images

### PR DESCRIPTION
**Pull request by davidcurrie**
Corresponds to https://github.com/docker-library/official-images/pull/958

### CODE REVIEW

**Comment by yosifkit** [comment]
We updated all the docs to work better with the new hub so it looks like this needs a rebase.  The new hub renders indent code blocks funky so we have moved to fenced blocks with a language defined (which is probably where the merge conflict is).  The content looks good to me :+1:.

**Comment by davidcurrie** [comment]
Rebased to pull in code blocks - should merge cleanly now.

File: websphere-liberty/content.md [hunk 1]
```diff
@@ -1,23 +1,41 @@
...
+
+# Images
+
+There are multiple images available in this repository. The image with the tag `beta` contains the contents of the runtime JAR for the latest monthly beta. The other images are all based on the latest generally available fix pack.
+
+The `kernel` image contains just the Liberty kernel and no additional runtime features. This image can be used as the basis for custom built images that contain only the features required for a specific application. For example, the following Dockerfile starts with this image, copies in the `server.xml` that lists the features required by the application, and then uses the `installUtility` command to download those features from the online repository.
+
+	FROM websphere-liberty:kernel
+	COPY server.xml /opt/ibm/wlp/usr/servers/defaultServer/
+	RUN installUtility install --acceptLicense defaultServer
```
Comment (yosifkit) [inline websphere-liberty/content.md]: Might want to make this a fenced block with a language so that the Docker Hub renders it better.

**Comment by davidcurrie** [comment]
Thanks @yosifkit - updated with additional fenced code block.

**Comment by yosifkit** [comment]
LGTM, pending https://github.com/docker-library/official-images/pull/958

**Comment by tianon** [comment]
LGTM

**Pull request merged**